### ISSUE TITLE
UI issues in Device.Interfaces view in new UI

### ISSUE THREAD

**Issue by mzbroch** [nautobot/nautobot#5030]
### Environment
* Nautobot version (Docker tag too if applicable): 08c29079eac9 (v2.1.0)
* Python version:
* Database platform, version:
* Middleware(s):

### Steps to Reproduce
1. Visit https://demo.nautobot.com/dcim/devices/37a938b0-bd5a-4c25-9d98-a51c75d15ba9/interfaces/  (ams01-edge-01 interfaces in demo instance)
2.
3.

### Expected Behavior
1. Cable ID is shown in 4 rows - I really don't think Cable ID is meaningful to most of users - I'd rather present canonical label / or shorten the ID to minimum and present as first letters only: `7c7b (...)` 

2. Table should fit into display size

### Observed Behavior
1. Table goes beyond size
2. Rows are too high

<img width="1440" alt="Screenshot 2024-01-02 at 12 00 25" src="https://github.com/nautobot/nautobot/assets/16935122/0807d3c2-d43c-4254-be94-14daa134d6e6">
<img width="1440" alt="Screenshot 2024-01-02 at 12 00 34" src="https://github.com/nautobot/nautobot/assets/16935122/389c29f1-fd38-47df-9ae7-c2eb79ae2725">

**Comment by gsnider2195** [comment]
We have a few places where the table is overflowing. See also https://demo.nautobot.com/ipam/ip-addresses/000ff5c9-aadf-440f-a517-6c40f2676991/?tab=main

![image](https://github.com/nautobot/nautobot/assets/75227981/91ced62d-47d3-4706-bf15-b2f0fd9421c5)

**Comment by glennmatthews** [comment]
In the "Parent Prefixes" case, I'd vote for hiding the `Namespace` and `Description` columns as a start.

**Comment by glennmatthews** [comment]
In the Interfaces case, perhaps we could replace the Cable string with just a linked button? I'd think that in most cases the question is not "Which cable is attached to this interface?" but rather "Is there a cable attached to this interface or not?"

**Comment by glennmatthews** [comment]
IMHO we should also move away from "Description" being a default-visible column in most tables.

**Comment by mzbroch** [comment]
> In the "Parent Prefixes" case, I'd vote for hiding the `Namespace` and `Description` columns as a start.

I would say, it's 

> IMHO we should also move away from "Description" being a default-visible column in most tables.



> In the Interfaces case, perhaps we could replace the Cable string with just a linked button? I'd think that in most cases the question is not "Which cable is attached to this interface?" but rather "Is there a cable attached to this interface or not?"

**Comment by mzbroch** [comment]
> In the "Parent Prefixes" case, I'd vote for hiding the `Namespace` and `Description` columns as a start.
> IMHO we should also move away from "Description" being a default-visible column in most tables.

From operators point of view, "description" is very important - i.e.:
- Ops can easily lookup a prefix
- Ops can understand prefix role/function/ etc (without standardised use of tags/CFs/etc)

For the Namespace, this is mandatory field since 2.0 and removing it can be highly confusing if users have overlapping spaces.

> In the Interfaces case, perhaps we could replace the Cable string with just a linked button? I'd think that in most cases the question is not "Which cable is attached to this interface?" but rather "Is there a cable attached to this interface or not?"

I think this makes sense

**Comment by glennmatthews** [comment]
> For the Namespace, this is mandatory field since 2.0 and removing it can be highly confusing if users have overlapping spaces.

Yes, but the parent prefixes are *always* in the same namespace as the prefix currently being viewed, are they not? So this is redundant information?

**Comment by glennmatthews** [comment]
We need to take a broader look at why in 2.1 so many tables are now overflowing their wrappers. Suspect there's probably a single point somewhere in our CSS that needs correcting. @norbert-mieczkowski-codilime can you perhaps look into this?

**Comment by gsnider2195** [comment]
I was able to resolve this by wrapping the table in a `<div class="table-responsive">`. On https://demo.nautobot.com/ipam/ip-addresses/000ff5c9-aadf-440f-a517-6c40f2676991/?tab=main

## Before

![image](https://github.com/nautobot/nautobot/assets/75227981/aebc2a35-3d18-497a-96ca-3708e6073496)

## After

![image](https://github.com/nautobot/nautobot/assets/75227981/1eb9898c-6551-4b78-96bc-268aa510b6e9)

**Comment by gsnider2195** [comment]
This is however causing other issues with the page layout:

## Before

![image](https://github.com/nautobot/nautobot/assets/75227981/a113ab69-fec1-4926-8174-cc0ed240bae2)

## After

![image](https://github.com/nautobot/nautobot/assets/75227981/e129d94d-5fa6-4268-b766-1cc671a57c99)

**Comment by mzbroch** [comment]
This seems to be fixed as of Nov-2025
